Middleware: Add Custom Headers to HTTP responses (#59018)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

<hr />

## [server.custom_response_headers]

> Lets you specify additional headers that the server can add to HTTP(S) responses.

Reviewer: ```suggestion
This setting enables you to specify additional headers that the server adds to HTTP(S) responses.
```

> ```
> ; exampleHeader1 = exampleValue1
> ; exampleHeader2 = exampleValue2
> ```

Reviewer 2: I'm not sure a commented-out example is that useful in the doc. Maybe remove it, or replace with an uncommented example?

```suggestion
```
